external-crates/move/crates/move-package/src/lib.rs

```diff
@@ -103,6 +103,10 @@ pub struct BuildConfig {
     /// If `true`, disable linters
     #[clap(long, global = true)]
     pub no_lint: bool,
+
+    /// If `true`, enables extra linters
+    #[clap(long, global = true)]
+    pub lint: bool,
```

Comment:
I'm sure it will be an annoying bit of work, but can we combine `no_lint` and `lint` into a single field on the build options?

https://docs.rs/clap/latest/clap/_derive/_tutorial/chapter_3/index.html#argument-relations Relevant documentation for Clap, though it's a bit of a chore and likely means we'll need an impl block for the Lint parse options.

Reply:
Fine with it being marked as a TODO?

Reply:
Did some strange shit, but no idea how to test it?

Reply:
Or rather, I wnat to throw a test into sui somewhere but cant find the spot

Reply:
Happy with just a TODO,  but it looks like you did it anyway 🎉